base/associative.jl

```diff
@@ -121,6 +127,14 @@ julia> collect(values(a))
 """
 values(a::Associative) = ValueIterator(a)
 
+"""
+    pairs(collection)
+
+Return an iterator over `(index, value)` pairs for any
+collection that maps a set of indices to a set of values.
```

Comment:
Why speak about "indices" here, but of "keys" in the docstrings for `keys` and `values`?

Reply:
Good point, I'll clarify the doc strings.